tendrl/gluster_integration/manager/__init__.py

```diff
@@ -34,10 +34,12 @@ def main():
     NS.state_sync_thread = sds_sync.GlusterIntegrationSdsSyncStateThread()
 
     NS.node_context.save()
-    NS.tendrl_context.save()
-    NS.gluster_integration.definitions.save()
-    NS.gluster_integration.config.save()
+    NS.gluster.tendrl_context.save()
```

Comment:
I agree that using `NS.node_context.node_id` we can load `tendrl_context` and get the `integration_id` from the same.
But the issue here is that after re-start even the field `nodes/{node-id}/TendrlContext/integration_id` goes blank if no create/import executed after re-start. So effectively even if we use `NS.node_context.node_id` to load the `tendrl_context` its not going to help.
@r0h4n here we need to fix the node-agent re-start and it should populate the previously created `tendrl_context` in NS